docs/BUILDING.md

```diff
@@ -209,6 +202,19 @@ cd build-cmake
 cpack
 ```
 
+### Additional CMake Targets
+#### Clean
+```bash
+# If you need to clean the project you can run
+cmake --build build-cmake --target clean
+```
+
+#### Regenerate Asset Headers
```

Comment:
You removed the reference of `ExtractAssetHeaders` from the Linux and Windows sections, and now it looks like the target only belongs to Mac.

Should probably unify this across each platform, or maybe the "Additional CMake Targets" block should become a separate section not tied to a platform (with a note for specifying the correct cmake command path)